packages/output/src/electron-node/console-logger-server.ts

```diff
@@ -0,0 +1,83 @@
+/********************************************************************************
+ * Copyright (C) 2017 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable, postConstruct } from 'inversify';
+import { LoggerWatcher } from '@theia/core/lib/common/logger-watcher';
+import { LogLevelCliContribution } from '@theia/core/lib/node/logger-cli-contribution';
+import { ILoggerServer, ILoggerClient, ConsoleLogger } from '@theia/core/lib/common/logger-protocol';
+import { ChannelServiceImpl, LogOutputChannel } from '../node/channel-server';
+
+@injectable()
+export class ConsoleLoggerServer implements ILoggerServer {
```

Comment:
please give unique name to classes and files in repo to easy navigation, i.e. `OutputChannelLoggerServer`, check project conventions for naming